che-incubator/che-code
# Fix an automatic rebase against the upstream

### What does this PR do?
Fixes an auto-rebase against the upstream.

### What issues does this PR fix?

fixes https://github.com/eclipse/che/issues/22161

### How to test this PR?
1. All the PR status checks must be green.
2. Start a new workspace from [this test project](https://github.com/azatsarynnyy/java-spring-petclinic/tree/fix-rebase) (with the editor built from the current PR) and verify that the extensions recommendations [auto-installation feature](https://github.com/che-incubator/che-code/pull/153) works as expected.
